core/validation/casting.py

```diff
@@ -26,6 +26,11 @@ def cast_to_schema(data: dict[str, pd.DataFrame], schema: dict) -> None:
                 original_type = table_types[column]
 
                 if original_type != target_type:
+                    if target_type == "list":
+                        # Forcible conversion of values to target_type == # "list" occurs in the relevant
+                        # extraction function, and .astype does not support lists.
+                        continue
```

Comment:
This can be now updated to:

```# Forcible conversion of values to target_type == "list" occurs in extract_postcodes() and .astype does not support lists```